document/web-api/index.bs

```diff
@@ -118,7 +119,9 @@ Note: This algorithm accepts a {{Response}} object, or a
     1. Let |returnValue| be [=a new promise=]
     1. [=Upon fulfillment=] of |source| with value |unwrappedSource|:
         1. Let |response| be |unwrappedSource|'s [=Response/response=].
-        1. Let |mimeType| be the result of [=extracting a MIME type=] from |response|'s [=response/header list=].
+        1. Let |mimeType| be the result of [=getting=] `` `Content-Type` `` from |response|'s [=response/header list=].
+        1. Remove all [=HTTP tab or space=] from the start and end of |mimeType|.
```

Comment:
What if _mimeType_ is null?

Reply:
Also, if it's not null it's a byte sequence...

Reply:
Also, header values should not have leading or trailing HTTP tabs or spaces, since those [make the header value invalid](https://fetch.spec.whatwg.org/#header-value), and are stripped when appending from the `Headers` interface.

Reply:
That happens when you set the values, but that's not necessarily true for something coming from the server. Stripping as OP did is correct. Just need to strip bytes and not code points.